### ISSUE TITLE
improvements

### ISSUE THREAD

**Issue by bozoh** [bozoh/moodle-mod_simplecertificate#121]
Hello Carlos,

Thank you for your plugin.  It's working great!  

I wanted to ask you if there is a way to make a couple of additions to it.

1) The possibilty to delete certificates manually so they can be re-issued.  Or... the capability to edit name. This is because some users don't realise that their name when they register is going to be printed in the certificate so sometimes they enter everything in lower case.  I can edit their names in the system, but the certificate doesn't change when they click on it again.

2) Maybe also it would be useful to add a filter at the top  "name / surname" (same as in reports/Course completion) so we can find users easily, instead of having to browse pages.

Thank you for your feedback.

**Comment by goose2000** [comment]
Just following. I often have to re-issue because of names spelled wrong or HR person enters the name wrong. A couple times learner got married and had a name change request.

Thank you Carlos; it's the best certificate module ever!

**Comment by heather-williams** [comment]
Yes, if you could make it so admins/teachers have the ability to set haschange = 1 via the UI that would be great!
